tools/extras/check_dependencies.sh

```diff
@@ -87,28 +87,40 @@ if ! which awk >&/dev/null; then
   add_packages gawk gawk gawk
 fi
 
-if which python >&/dev/null ; then
+pythonok=true
+if ! which python2.7 >&/dev/null; then
+  echo "$0: python2.7 is not installed"
+  add_packages python2.7
+  pythonok=false
+fi
+
+if ! which python3 >&/dev/null; then
+  echo "$0: python3 is not installed"
+  add_packages python3
+  pythonok=false
```

Comment:
I see you are adding requirement for python3 -- is this necessary? I remeber @danpovey has a couple of scripts for LM stuff, but those were not merged yet.